### PR TITLE
Revert "Replaced runOnUiThread with Handlers, cleaned up code"

### PR DESCRIPTION
**Pull request by clarisaleu**
Reverts footprnt/footprnt#121

please fix the toolbar and fix app from crashing

### CODE REVIEW

**Review by jocelynshen — APPROVED** [review]

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java [hunk 1]
```diff
@@ -36,24 +34,26 @@ public ListAdapter(Context context, ArrayList<Business> businesses) {
         mContext = context;
     }
 
+
     @NonNull
     @Override
     public ListAdapter.BusinessViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_business, parent, false);
-        return new BusinessViewHolder(view);
+        BusinessViewHolder viewHolder = new BusinessViewHolder(view);
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:42]: Why not simplify this to
  return new BusinessViewHolder(view);
??

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java [hunk 2]
```diff
@@ -36,24 +34,26 @@ public ListAdapter(Context context, ArrayList<Business> businesses) {
...
+        return viewHolder;
     }
 
     @Override
     public void onBindViewHolder(ListAdapter.BusinessViewHolder holder, int position) {
         holder.bindBusiness(mBusinesses.get(position));
     }
 
+
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:51]: Why the extra empty line?

File: app/src/main/java/com/example/footprnt/Discover/Services/YelpService.java [hunk 3]
```diff
@@ -26,23 +26,25 @@
...
-     * Finds <query> in <location>
+     * Finds <query> in <location><
      *
      * @param location location to query in
      * @param query    type of query to make
      * @param callback response from yelp
      */
-
     public static void findBusinesses(String location, String query, Callback callback) {
+
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Services/YelpService.java:41]: Why all the empty lines?

File: app/src/main/java/com/example/footprnt/Discover/Services/YelpService.java [hunk 4]
```diff
@@ -26,23 +26,25 @@
...
+
         OkHttpClient client = new OkHttpClient.Builder()
                 .build();
 
         HttpUrl.Builder urlBuilder = HttpUrl.parse(Constants.YELP_BASE_URL + location + "&term=" + query).newBuilder();
+        //urlBuilder.addQueryParameter(Constants.YELP_LOCATION_QUERY_PARAMETER, location);
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/Services/YelpService.java:47]: No commented out code in the master branch

File: app/src/main/java/com/example/footprnt/Discover/ViewHolders/RestaurantView.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package com.example.footprnt.Discover.ViewHolders;
+
+import android.content.Context;
+import android.view.View;
+
+public class RestaurantView extends View {
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/Discover/ViewHolders/RestaurantView.java:6]: Javadoc please :)

File: app/src/main/java/com/example/footprnt/HomeActivity.java [hunk 6]
```diff
@@ -69,6 +69,7 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 mp.start();
+                Fragment fragment;
```
Comment (ajankenm) [inline app/src/main/java/com/example/footprnt/HomeActivity.java:72]: This is dead code

File: app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java (hunk 1 above)
Comment (stanleynwakamma) [reply app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java:42]: Checking my commit, I corrected it to "return new BusinessViewHolder (View)" However my commit was Reverted by Claire

**Comment by stanleynwakamma** [comment]
This is a reverted commit. I made most changes in the original commit.

**Pull request merged**